File: recipes/java-recipes/quarkus-recipe/src/main/java/com/trendyol/stove/recipes/quarkus/QuarkusMainApp.java
package com.trendyol.stove.recipes.quarkus;

import io.quarkus.runtime.Quarkus;
import io.quarkus.runtime.QuarkusApplication;
import io.quarkus.runtime.annotations.QuarkusMain;

@QuarkusMain
public class QuarkusMainApp {
  public static void main(String[] args) {
    Quarkus.run(QuarkusApp.class, args);
  }

  public static class QuarkusApp implements QuarkusApplication {

    @Override
    public int run(String... args) {
      Quarkus.waitForExit();
      return 0;
    }
  }
}

File: examples/quarkus-example/src/main/kotlin/stove/quarkus/example/QuarkusMainApp.kt
package stove.quarkus.example

import io.quarkus.runtime.Quarkus
import io.quarkus.runtime.annotations.QuarkusMain

@QuarkusMain
object QuarkusMainApp {
  @JvmStatic
  fun main(args: Array<String>) {
    Quarkus.run(*args)
  }
}
